sortfilterproxymodel.qbs

```diff
@@ -1,12 +1,14 @@
 import qbs 1.0
+import qbs.File
+import qbs.FileInfo
 
 Product
 {
-    name: "sortfilterproxymodel"
-    type: "dynamiclibrary"
+    name: "SortFilterProxyModel"
+    type: ["dynamiclibrary", "copied qbs resources"]
```

Comment:
@johnbivens So the way qbs works is via. fileTags. "copied qbs resources" is the fileTags for the artifacts described here: 
https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/sortfilterproxymodel.qbs#L101

Also, the place i got stuck, trying to figure out why this rule wasn't getting executed was:
I was using input: ["qmldir"] instead of:      https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/sortfilterproxymodel.qbs#L95 , and tagging the group of qbs resources with: 
https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/sortfilterproxymodel.qbs#L89